scripts/build.sh

```diff
@@ -72,6 +72,12 @@ function RunXcodebuild() {
   # Workaround simulator flake introduced with Xcode 10.1.
   # TODO: Investigate performance implications and impact of only resetting
   #       necessary devices and skipping for macOS.
+  killall Simulator >/dev/null 2>&1 || echo "No Simulator.app running."
```

Comment:
This is getting out of hand. Is there a doc somewhere you're following for ideas about this? If so, could you link here?

Alternatively, could you comment a bit on each one grouping what you're trying to do here? I worry that in six months time we'll have no idea what this black magic does and we'll never be able to get rid of it.

Reply:
I've been experimenting to find the relevant pieces of https://github.com/material-foundation/ios-simulator-utils/blob/master/scripts/cleanup_simulators.sh which is at least a part of a solution on internal infrastructure. It doesn't include the code to recreate new simulators.

So far, it seems that killing, clearing and restarting simulators isn't sufficient. 

Next, I'll try the recommendation of Travis support of `travis_retry` and then investigate creating new simulators for each run. 

Once something works, I'll send another PR to make the documentation clear.